src/schema/portal_user_facility.yaml

```diff
@@ -0,0 +1,163 @@
+name: user_facility_metadata
+description: This file defines terms that appear in the 'JGI-Metagenomics', `JGI-Metagenomics Long Read`, `JGI-Metatranscriptomics`, `EMSL` sections
+  of the NMDC sample metadata submission portal, which is implemented with DataHarmonizer
+  as of Spring 2022
+title: 'NMDC Schema supplement for Submission Portal: User Facility Metadata'
+id: https://w3id.org/nmdc/portal/jgi_metagenomics
+
+license: https://creativecommons.org/publicdomain/zero/1.0/
+
+imports:
+  - portal_enums
+  - nmdc_types
+
+prefixes:
+  linkml: "https://w3id.org/linkml/"
+  nmdc: "https://w3id.org/nmdc/"
+
+default_prefix: nmdc
+default_range: string
+
+slots:
+  rna_collect_site:
+    description: Provide information on the site your RNA sample was collected from
+    title: RNA collection site
+    examples:
+      - value: untreated pond water
+    rank: 15
+    string_serialization: '{text}'
+    slot_group: JGI-Metatranscriptomics
+    recommended: true
+  rna_isolate_meth:
+    description: Describe the method/protocol/kit used to extract DNA/RNA.
+    title: RNA isolation method
+    examples:
+      - value: phenol/chloroform extraction
+    aliases:
+      - Sample Isolation Method
+    rank: 16
+    string_serialization: '{text}'
+    slot_group: JGI-Metatranscriptomics
+    recommended: true
+  rna_organisms:
+    description: List any organisms known or suspected to grow in co-culture, as well
+      as estimated % of the organism in that culture.
+    title: RNA expected organisms
+    examples:
+      - value: expected to contain microbes (59%) fungi (30%), viruses (10%), tadpoles
+          (1%)
+    rank: 14
+    string_serialization: '{text}'
+    slot_group: JGI-Metatranscriptomics
+    recommended: true
+  dna_lr_isolate_meth:
+    description: Describe the method/protocol/kit used to extract DNA/RNA.
+    title: DNA isolation method for long read sequencing
+    examples:
+      - value: phenol/chloroform extraction
+    rank: 16
+    string_serialization: '{text}'
+    slot_group: JGI-Metagenomics
```

Comment:
@pkalita-lbl is this right? is there a separate slot group for the long read metagenomics in the submission portal? I don't think there is, but I'm not 100% on that.

And I think, once this is done, I'll need to update https://github.com/microbiomedata/submission-schema/blob/f8fd2a795b2ee006275e90272247bae20c47b4cd/src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml#L727